hydrosdk/cluster.py

```diff
@@ -45,7 +83,7 @@ def request(self, method, url, **kwargs):
         :param kwargs: additional args
         :return: request res
         """
-        url = parse.urljoin(self.address, url)
+        url = parse.urljoin(self.http_address, url)
         return requests.request(method, url, **kwargs)
 
     def grpc_secure(self, credentials=None, options=None, compression=None):
```

Comment:
This method is useless now, since the creation of channel is in the ctor. Is suppose we should delete them.

Reply:
Deleted